Logging/src/V2/Gapic/ConfigServiceV2GapicClient.php

```diff
@@ -1017,469 +2174,464 @@ public function deleteSink($sinkName, array $optionalArgs = [])
-     * @param FieldMask    $updateMask Required. A non-empty list of fields to change in the existing exclusion.
-     *                                 New values for the fields are taken from the corresponding fields in the
-     *                                 [LogExclusion][google.logging.v2.LogExclusion] included in this request. Fields not mentioned in
-     *                                 `update_mask` are not changed and are ignored in the request.
+     *                                   Example: `"organizations/12345/cmekSettings"`.
      *
-     * For example, to change the filter and description of an exclusion,
-     * specify an `update_mask` of `"filter,description"`.
-     * @param array $optionalArgs {
-     *                            Optional.
+     *                                   Note: CMEK for the Logs Router can currently only be configured for GCP
+     *                                   organizations. Once configured, it applies to all projects and folders in
+     *                                   the GCP organization.
+     * @param CmekSettings $cmekSettings Required. The CMEK settings to update.
      *
+     *                                   See [Enabling CMEK for Logs
+     *                                   Router](https://cloud.google.com/logging/docs/routing/managed-encryption)
+     *                                   for more information.
+     * @param array        $optionalArgs {
+     *     Optional.
+     *
+     *     @type FieldMask $updateMask
+     *           Optional. Field mask identifying which fields from `cmek_settings` should
+     *           be updated. A field will be overwritten if and only if it is in the update
+     *           mask. Output only fields cannot be updated.
+     *
+     *           See [FieldMask][google.protobuf.FieldMask] for more information.
+     *
+     *           Example: `"updateMask=kmsKeyName"`
      *     @type RetrySettings|array $retrySettings
-     *          Retry settings to use for this call. Can be a
-     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
-     *          of retry settings parameters. See the documentation on
-     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     *           Retry settings to use for this call. Can be a
+     *           {@see Google\ApiCore\RetrySettings} object, or an associative array of retry
+     *           settings parameters. See the documentation on
+     *           {@see Google\ApiCore\RetrySettings} for example usage.
      * }
      *
-     * @return \Google\Cloud\Logging\V2\LogExclusion
+     * @return \Google\Cloud\Logging\V2\CmekSettings
      *
      * @throws ApiException if the remote call fails
-     * @experimental
      */
-    public function updateExclusion($name, $exclusion, $updateMask, array $optionalArgs = [])
+    public function updateCmekSettings($name, $cmekSettings, array $optionalArgs = [])
```

Comment:
This method had both optional parameters `$name` and `$cmekSettings` moved to be required arguments.